android/src/main/java/com/tailscale/ipn/ui/viewModel/VpnViewModel.kt

```diff
@@ -29,6 +29,8 @@ class VpnViewModel(application: Application) : AndroidViewModel(application) {
   // Whether the VPN is prepared
```

Comment:
Can we provide comments to explain the difference between `vpnPrepared` and `vpnActive`, and their purpose? Who sets these, and who reads them?